seqr/views/utils/export_utils.py

```diff
@@ -64,25 +65,42 @@ def export_table(filename_prefix, header, rows, file_format='tsv', titlecase_hea
         raise ValueError("Invalid file_format: %s" % file_format)
 
 
-def export_multiple_files(files, zip_filename, file_format='csv', add_header_prefix=False, blank_value=''):
+def _format_files_content(files,  file_format='csv', add_header_prefix=False, blank_value=''):
     if file_format not in DELIMITERS:
         raise ValueError('Invalid file_format: {}'.format(file_format))
+    parsed_files = []
+    for filename, header, rows in files:
+        header_display = header
+        if add_header_prefix:
+            header_display = ['{}-{}'.format(str(header_tuple[0]).zfill(2), header_tuple[1]) for header_tuple in
+                              enumerate(header)]
+            header_display[0] = header[0]
+        content = DELIMITERS[file_format].join(header_display) + '\n'
+        content_rows = [[row.get(key) or blank_value for key in header] for row in rows]
+        content += '\n'.join([
+            DELIMITERS[file_format].join(row) for row in content_rows
+            if any(val != blank_value for val in row)
+        ])
+        content = str(content.encode('utf-8'), 'ascii', errors='ignore')  # Strip unicode chars in the content
+        parsed_files.append(('{}.{}'.format(filename, file_format), content))
+    return parsed_files
+
+
+def export_multiple_files(files, zip_filename, **kwargs):
```

Comment:
Maybe, the Anvil report should also be saved to a bucket so that no file downloading is needed for the reports.

Reply:
Tis ticket is about the Gregor reports which are delivered to the Gregor consortium via AnVIL workspaces so Sam put in a ticket to streamline that process, but the AnVIL reports are a totally different  thing that are used in a totally different way and noone asked us to move, so no